src/lib.rs

```diff
@@ -4,9 +4,6 @@
 
 #![cfg_attr(not(test), no_std)]
 
-use arrayvec::ArrayString;
```

Comment:
Does it not work to disable the std features in arrayvec i.e. make it no_std compatible?